src/main/java/spring/intro/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package spring.intro.controller;
+
+import dto.UserResponseDto;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import spring.intro.model.User;
+import spring.intro.service.UserMapper;
+import spring.intro.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private final UserService userService;
+    private UserMapper userMapper;
+
+    @Autowired
+    public UserController(UserService userService, UserMapper userMapper) {
+        this.userService = userService;
+        this.userMapper = userMapper;
+    }
+
+    @GetMapping("/inject")
+    public String inject() {
+        User bob = new User();
+        bob.setEmail("bob@gmail.com");
+        bob.setPassword("bobpass");
+        User alice = new User();
+        alice.setEmail("alice@gmail.com");
+        alice.setPassword("alicepass");
+        userService.add(bob);
+        userService.add(alice);
+        return "All users were injected";
```

Comment:
Optional. I don`t know was it a strong requirement to create exactly 4 users, so maybe 2 will be enough.